ci/testflight/fulcrum/main.tf

```diff
@@ -0,0 +1,105 @@
+variable "testflight_namespace" {}
+
+locals {
+  cluster_name     = "galoy-staging-cluster"
+  cluster_location = "us-east1"
+  gcp_project      = "galoy-staging"
+
+  smoketest_namespace  = "galoy-staging-smoketest"
+  testflight_namespace = var.testflight_namespace
+}
+
+resource "kubernetes_namespace" "testflight" {
+  metadata {
+    name = local.testflight_namespace
+  }
+}
+
+data "kubernetes_secret" "bitcoin_rpcpassword" {
+  metadata {
+    name      = "bitcoind-rpcpassword"
+    namespace = "galoy-staging-bitcoin"
+  }
+}
+
+resource "kubernetes_secret" "bitcoinrpc_password" {
```

Comment:
There's some naming inconsistency here. `bitcoinrpc_password` here vs `bitcoin_rpcpassword` in the data resource

Reply:
good catch. It is similar in the testflight galoy and lnd as well. Will unify them in a separate PR.

Reply:
done in https://github.com/GaloyMoney/charts/pull/2418/commits/7744447523be13e628ed0da4d3109fc4b6a3f96f